processor-kotlin/src/test/resources/schema/EnumItem.json

```
@@ -0,0 +1,7 @@
+{
+  "types" : [ {
+    "name" : "Status",
+    "values" : [ "RUNNING", "PAUSED", "STOPPED" ],
+    "type" : "ENUM"
+  } ]
+}
```

We should always end files with a line feed.
```suggestion
}

```

---

Agreed (usually), but those files are being used to compare them against compile outputs.. the Compiler output does not have a￼n empty line at the end.

I guess we should rather add a specific compare(file,file) function which just ignores empty lines for comparison. (Unfortunately haven't found any library with such a built-in function)